images/base/Dockerfile

```diff
@@ -27,15 +27,15 @@ ARG TARGETARCH
 
 # Configure containerd and runc binaries from kind-ci/containerd-nightlies repository
 # The repository contains latest stable releases and nightlies built for multiple architectures
-ARG CONTAINERD_VERSION="1.6.20"
+ARG CONTAINERD_VERSION="1.6.20-14-g967a516de"
 ARG CONTAINERD_BASE_URL="https://github.com/kind-ci/containerd-nightlies/releases/download"
 ARG CONTAINERD_URL="${CONTAINERD_BASE_URL}/containerd-${CONTAINERD_VERSION}/containerd-${CONTAINERD_VERSION}-linux-${TARGETARCH}.tar.gz"
-ARG CONTAINERD_AMD64_SHA256SUM="a6432bbdb0573887a6b2b91d8a1e2cc00c97c59b6210dbacd52919a4c815b5b2"
-ARG CONTAINERD_ARM64_SHA256SUM="16014836cbd93f833d4f58c9d02952d373bcc4c9f8e3b0ad778d625106264cca"
+ARG CONTAINERD_AMD64_SHA256SUM="429140ca5c264fa2ddc34b5a5e894c45a4c1dc0e5c13a5d18012878b76b89d88"
+ARG CONTAINERD_ARM64_SHA256SUM="be56d1583f627b0e7afefd2d9ee400f181c646b9d729217d986fa8ed7b8745ad"
 
 ARG RUNC_URL="${CONTAINERD_BASE_URL}/containerd-${CONTAINERD_VERSION}/runc.${TARGETARCH}"
-ARG RUNC_AMD64_SHA256SUM="415d6e654754b51e907088cea751da45784abcba748938636471d3016e546385"
-ARG RUNC_ARM64_SHA256SUM="6c37b993e2a14bcfe18789ab0e80d0ce5f24e19aa95ba8607b1152f7bd6bceca"
+ARG RUNC_AMD64_SHA256SUM="d78d68124ab6207c498d6fcaf527f5156832b307059e7daa97d64eb99a24162d"
+ARG RUNC_ARM64_SHA256SUM="82b27080bbef08649d49094ee03e504fbb60dea40d5f914e3a908264bc048af1"
```

Comment:
this should move move us to 1.1.6, the containerd commit is newer than the commit that moves 1.1.5